x-pack/plugins/uptime/common/constants/alerts.ts

```diff
@@ -7,23 +7,33 @@
 
 import { ActionGroup } from '../../../alerts/common';
 
+export type MonitorStatusActionGroup = ActionGroup<'xpack.uptime.alerts.actionGroups.monitorStatus'>;
+export type TLSActionGroup = ActionGroup<'xpack.uptime.alerts.actionGroups.tls'>;
+export type DurationAnomalyActionGroup = ActionGroup<'xpack.uptime.alerts.actionGroups.durationAnomaly'>;
+
+export const MONITOR_STATUS: MonitorStatusActionGroup = {
```

Comment:
The compiler seemed to want these exported individually in the way we were referencing them.